imagepuller/internal/service/verify_test.go

```diff
@@ -102,3 +118,280 @@ func TestStoreAndVerifyLayers_EvilRegistry(t *testing.T) {
 		})
 	}
 }
+
+var zeroDigest = "0000000000000000000000000000000000000000000000000000000000000000"
+
+type StubRemote struct {
+	mediaType  types.MediaType
+	imageIndex StubImageIndex
+
+	headErr  bool
+	imageErr bool
+	indexErr bool
```

Comment:
Here is where I meant to introduce configurable errors, like e.g.

https://github.com/edgelesssys/contrast/blob/0c18004dfc697745199fb1acaba646c7d0074a24/coordinator/internal/userapi/userapi_test.go#L927-L930